linera-service/src/linera/main.rs

```diff
@@ -1609,40 +1715,57 @@ async fn run(options: &ClientOptions) -> Result<i32, anyhow::Error> {
 
         ClientCommand::Wallet(wallet_command) => match wallet_command {
             WalletCommand::Show { chain_id, short } => {
+                let start_time = Instant::now();
                 if *short {
                     for chain_id in options.wallet().await?.chains.keys() {
                         println!("{chain_id}");
                     }
                 } else {
                     wallet::pretty_print(&*options.wallet().await?, *chain_id);
                 }
+                info!("Showing wallet in {} ms", start_time.elapsed().as_millis());
                 Ok(0)
             }
 
             WalletCommand::SetDefault { chain_id } => {
+                let start_time = Instant::now();
                 options
                     .wallet()
                     .await?
                     .mutate(|w| w.set_default_chain(*chain_id))
                     .await??;
+                info!(
+                    "Setting default chain in {} ms",
```

Comment:
That's not the best sentence in English (it should be in the past)

Reply:
Ok, corrected. That is indeed important as it is visible to users.